metabase/metabase
# Metabase Entity Link Dashboard Cards

Resolves https://github.com/metabase/metabase/issues/28246

Backend: https://github.com/metabase/metabase/pull/28297

### Description

- adds global entity search to link cards and allows them to be added to link cards with the richer previews we use for search

![entities](https://user-images.githubusercontent.com/30528226/219531322-5752a8f5-4c90-4f5c-a522-995de596f5e7.gif)


### How to verify

- spin up the backend from https://github.com/metabase/metabase/pull/28297
- Open a dashboard and click edit
- click the link button to add a link dashcard
- start typing in the search box and you should get a popover with various metabase entities. 
- Selecting an entity should populate it in the dashcard
- clicking the edit button should allow you to edit the entity
- any search query starting in "http" should make the search box go away (so users don't get annoying search results when they're just pasting an external link)

### Checklist
